src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double firstBase;
+    private double secondBase;
+    private double height;
+    private String color;
```

Comment:
remove String color from every type of figure

Reply:
Why?

Reply:
if I remove it, I will have to change the all code